tests/ci_build/Dockerfile.gpu

```diff
@@ -18,7 +18,7 @@ ENV PATH=/opt/python/bin:$PATH
 RUN \
     pip install numpy pytest scipy scikit-learn pandas matplotlib wheel kubernetes urllib3 graphviz && \
     pip install "dask[complete]" && \
-    conda install -c rapidsai -c nvidia -c numba -c conda-forge -c anaconda dask-cuda
+    conda install -c rapidsai -c conda-forge -c anaconda dask-cuda
```

Comment:
I don't think this line is necessary for running your test.

Reply:
Never mind. I found issues with the dockerfile that was causing Docker images to fail.